src/main/Ticket.java

```diff
@@ -105,6 +87,75 @@ public String toString() {
                 ", stadiumSector=" + sector +
                 ", maxBackpackWeightInKg=" + maxBackpackWeightInKg +
                 ", creationTime=" + formattedCreationTime +
+                ", price=" + price +
                 '}';
     }
+
+    @Override
+    public String toString() {
+        return getAllTicketValues();
```

Comment:
why we create getAllTicketValues()? We can just use to String with the same logic)

Reply:
In our homework (4) We have task 3: There should be a possibility to get all Ticket values. So I made a method getAllTicketValues () that returns String with all ticket values. Maybe  in this case it would be better to return Collection with all ticket values?